protokube/cmd/protokube/main.go

```diff
@@ -188,7 +199,7 @@ func run() error {
 	}
 
 	rootfs := "/"
-	if containerized {
+	if containerized && cloud != "vsphere" {
```

Comment:
Just wondering what's this containerized means and why vsphere should not set to /rootfs/ ?